src/OrbitGl/App.cpp

```diff
@@ -567,6 +567,23 @@ void OrbitApp::OnErrorEnablingOrbitApiEvent(
       });
 }
 
+void OrbitApp::OnErrorEnablingUserSpaceInstrumentationEvent(
+    orbit_grpc_protos::ErrorEnablingUserSpaceInstrumentationEvent error_event) {
+  main_thread_executor_->Schedule([this, error_event = std::move(error_event)]() {
+    main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
+                                     GetCaptureTimeAt(error_event.timestamp_ns()),
+                                     error_event.message());
+
+    if (!IsLoadingCapture()) {
+      constexpr const char* kDontShowAgainErrorEnablingUserSpaceInstrumentationWarningKey =
+          "DontShowAgainErrorEnablingUserSpaceInstrumentationWarning";
+      main_window_->ShowWarningWithDontShowAgainCheckboxIfNeeded(
```

Comment:
Do we even want users to be able to disable showing this warning in the future? This seems such a severe case that it should always be shown. 

I'm also wondering if we should say more about what users can do about this (which is probably nothing?), or is there more in the message itself?

Reply:
Yep, I hesitated when writing the DontShowAgain-part. I'm fine with removing it. Let's see if @dpallotti has an opinion.

There is one case where the user can do something: If there is a debugger attached you can remove it and the error message should mention that. Besides that it forwards error messages from "stuff that should never fail"<sup>_tm_</sup>.

Reply:
We also allow the other severe warnings to not be shown again as message boxes. The reason is that those warnings show up in orange in the capture log and they are very visible even without the message box. So the idea is that it's better like this than to have the message box come up every time in case something goes wrong every time.

Reply:
Okay, thanks, this helps! So since this will not be lost I'll keep the DontShowAgain checkbox.

Reply:
It's the `main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,` above.

Reply:
Okay, sounds good to me regarding showing severe issues as orange in the capture log.